custom_components/trakt_tv/sensor.py

```diff
@@ -51,6 +51,21 @@ async def async_setup_entry(hass, config_entry, async_add_entities):
             )
             sensors.append(sensor)
 
+        # Ensure anticipated sensors are added
```

Comment:
You've nested it under the first loop which stops if not a BASIC KIND (see line 40), which is why no sensors are created.
It should be separated out at the end of the function
```

    for trakt_kind in TraktKind:
        if trakt_kind not in ANTICIPATED_KINDS:
            continue

        identifier = trakt_kind.value.identifier

        if configuration.anticipated_identifier_exists(identifier):
            sensor = TraktSensor(
                hass=hass,
                config_entry=config_entry,
                coordinator=coordinator,
                trakt_kind=trakt_kind,
                source="anticipated",
                prefix="Trakt Anticipated",
                mdi_icon="mdi:movie" if trakt_kind == TraktKind.ANTICIPATED_MOVIE else "mdi:television",
            )
            sensors.append(sensor)
```